thrift.js

```diff
@@ -177,13 +180,13 @@ Thrift.prototype._asyncParse = function _asyncParse(filename, allowIncludeAlias,
         return model._asyncParseIncludedModules(filename, allowIncludeAlias, cb);
     }
 
-    model.fs.readFile(filename, function (err, source) {
+    model.fs.readFile(filename, this.encoding, function (err, source) {
```

Comment:
Is it normal that the encoding is passed as argument twice here?

Reply:
nope. ty.